### DISCUSSION TITLE
企業家夢 不 是明星夢

### DISCUSSION THREAD

**Discussion by Teddy111802518** [1101-GS4719-C/Python-Community-GS4719-C#2274] [General]
前陣子偶然在網路上畫到一部黃仁勳的科技發表會的影片，畫面裡大家聚集在體育館裏面 超級多人 像是一場盛大的演唱會，你坐在後面根本看不到黃仁勳，標題就一下，黃仁勳是男人世界的Taylor Swift，我一下腦袋就炸開了，我怎麼會忽略掉自己的信仰呢，只要受人追捧指引他人，人人都能是信仰，黃仁勳簡直就像是明星在舞台上，真的是讓我更確信要創業哈哈，聽起來好廉價，但創業點子確實有這個程度，能產生藥物開發的革命

**Comment by Teddy111802518** [comment]
哈哈我只能說我長得不高不帥不可能站在舞台上當明星，但是這讓我發現新的方式，以後我產品發布會，來參加的我送你們公司標誌的首燈啦，哈哈哈我一定要搞得像是演唱會一樣，讓大家慕名而來，其實也是個很錯的推銷呢

**Comment by christaaah** [comment]
我覺得他真的已經像是偶像一樣了，去到一個學校演講後那個學校的報名率就爆滿，某種程度來說這種行為就好像追星。但我覺得這種關乎到自己人生的大事的事情還是再三思考比較好，多方比較再決定比較穩當。
